docs/swagger.json

```diff
@@ -72,15 +69,12 @@
         ],
         "summary": "Deletes or revokes CA from Vault.",
```

Comment:
Vault is just the current backend for CA storage. It is better to keep the API doc agnostic from the current backend implentation.